rose/server/game.py

```diff
@@ -56,6 +56,7 @@ def stop(self):
 
     def add_player(self, name):
         if name in self.players:
+            print 'Failed to add player:', name, 'player already exists'
```

Comment:
This is not the place to fix this - please print the exception in lineReceived, so see all errors, not only this specific error.